pkg/apis/k0s/v1beta1/api.go

```diff
@@ -72,6 +73,17 @@ func DefaultAPISpec() *APISpec {
 	return a
 }
 
+func (a *APISpec) LocalURL() *url.URL {
+	var host string
+	if a.OnlyBindToAddress {
+		host = net.JoinHostPort(a.Address, strconv.Itoa(a.Port))
+	} else {
+		host = fmt.Sprintf("localhost:%d", a.Port)
+	}
+
+	return &url.URL{Scheme: "https", Host: host}
```

Comment:
It doesn't matter, but....
```suggestion
	host := "localhost"
	if a.OnlyBindToAddress {
		host = a.Address
	}
	return &url.URL{Scheme: "https", Host: net.JoinHostPort(host, strconv.Itoa(a.Port))}
```